app/api/reminders/route.ts

```diff
@@ -0,0 +1,124 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { prisma } from '@utils/prisma';
+
+// Configuration constants
+const UPLOAD_LINK = 'https://sistema-prod.vercel.app/calendar';
+const ADMIN_EMAIL = 'admin@sistema.ca';
+const EMAIL_API_ENDPOINT = 'http://localhost:3000/api/sendEmail';
```

Comment:
This will not work when moved to prod. You could either check the environment (prod or dev) and change the base url (http://localhost:3000/api/ vs our prod url) accordingly or you can move the email sending logic to a regular .ts file rather than an API endpoint. I went with the latter.